src/local-editor-service.ts

```diff
@@ -98,21 +99,55 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModule =
+          this.getAncestorDomModuleForElement(document, location.element);
+      if (!domModule || !domModule.id)
+        return;
+      const outerElement = document.getOnlyAtId('element', domModule.id);
+      if (!outerElement)
+        return;
+      const sortPrefixes = this._createSortPrefixes(outerElement);
+      const innerElement =
+          document.getOnlyAtId('element', location.element.nodeName)!;
+      const innerAttribute = innerElement.attributes.find(
+          (value) => value.name === location.attribute)!;
+      const attributeValue =
+          dom5.getAttribute(location.element, innerAttribute.name)!;
+      const hasDelimeters = /^\s*(\{\{|\[\[)/.test(attributeValue);
+      return {
+        kind: 'attribute-values',
+        attributes: outerElement.attributes.map(p => {
```

Comment:
I think we want to use `outerElement.properties` here, not attributes, as it data-binds the value of the property to the inner element (and I think if we do that we'll need to convert kebab-case to snakeCase).

For real pedantic correctness, we could access the inner attribute if its name was suffixed with a `$`, and the inner property otherwise. This doesn't matter much for Polymer elements, so this is fine to land as is for now, but if we're able to get the docs for all native elements in here then there's a bunch of places where attributes and properties diverge.